configure.ac

```diff
@@ -46,5 +46,12 @@ AC_TRY_LINK([
 AC_DEFINE(HAVE_FULL_QUEUE_H,, Define if sys/queue.h includes required macros)
 ])
 
+AC_TRY_LINK([
+	#include <ncursesw/curses.h>
+	#include <ncursesw/term.h>
+], [], [
+AC_DEFINE(HAVE_NCURSESW_H,, Define if ncursesw is available)
+])
```

Comment:
@eavgerinos No stress! Thanks for fixing it!

I'm still curious, though, what does the `AC_TRY_LINK` construct do that the `AC_SEARCH_LIBS` and `config.h.in` does not?